### ISSUE TITLE
Hard links are not set up correctly

### ISSUE THREAD

**Issue by puresilk** [jrobinson-uk/Pi-Noobs-Converter#2]
You were wondering why the new image needs a bit more space - this is probably because the hard links in the original image are not set up correctly.

Tested with Raspbian and your script

-> in the NOOBSified image there are no hardlinks at all, in the original there are:
e.g. /usr/share/doc/libcurl3/BUGS.gz

I used instructions from here http://stackoverflow.com/questions/1810334/how-to-find-all-files-which-are-basically-soft-or-hard-links-of-other-directorie

Basically to search for soft links, as user root in the root directory:

 find . -type l | less

to search for hard links:
find . -type f -links +1 | less

I also suspect that the amount of softlinks changes (they probably become less) - but this is not proven yet.

**Comment by puresilk** [comment]
Might be fixable with -a option, I will test later:

http://unix.stackexchange.com/questions/44247/how-to-copy-directories-with-preserving-hardlinks

**Comment by puresilk** [comment]
Yes, it is. Here's how to fix:
 sudo cp -a /tmp/Noobs-Converter/Orig/1/\* /tmp/Noobs-Converter/New/1/
 sudo cp -a /tmp/Noobs-Converter/Orig/2/\* /tmp/Noobs-Converter/New/2/

The -a option includes R.

Another suggestion: use the -v option on xz - it will show the progress in a nice way. On my equipment, for instance, the compression takes about one hour.
